common/protob/messages-ethereum.proto

```diff
@@ -160,6 +165,7 @@ message EthereumSignTypedHash {
     repeated uint32 address_n = 1;              // BIP-32 path to derive the key from master node
     required bytes domain_separator_hash = 2;   // Hash of domainSeparator of typed data to be signed
     optional bytes message_hash = 3;            // Hash of the data of typed data to be signed (empty if domain-only data)
+    optional bytes encoded_network = 4;         // encoded Ethereum network, see protocol.md for details
```

Comment:
Should probably refer to `docs/common/ethereum-definitions.md` instead.

Reply:
[864ab42](https://github.com/trezor/trezor-firmware/pull/2410/commits/864ab42333e99a6b93de2707e299ef3f3fca06e6)